event_contact/models/event.py

```diff
@@ -8,9 +8,9 @@
 class event_event(models.Model):
     _inherit = 'event.event'
 
-    contact_person_ids = fields.Many2many(
-        string='Contact Persons',
+    contacts_ids = fields.Many2many(
+        string='Contacts',
         comodel_name='res.partner',
-        relation='event_contact_person_rel',
+        relation='event_contacts_rel',
```

Comment:
Do you really need to specify the relation table? Is it not enough with the automatic name?